dev/setup_localstack.py

```diff
@@ -0,0 +1,192 @@
+        return arn
+
+    def create_security_group(self, group_name):
+        print(f"Creating security group {group_name}")
+        self.ec2_client.create_security_group(
+            Description=EC2_SECURITY_GROUP_DESCRIPTION,
+            GroupName=group_name
+        )
+        print("Created security group")
+
+    def get_security_group_id(self):
+        print("Getting security group ID")
+        response = self.ec2_client.describe_security_groups()
+        security_groups = response.get("SecurityGroups", [""])
+        group_id = security_groups[0].get("GroupId", "")
+        print(f"Got group ID {group_id}")
+        return group_id
+
+    def create_subnet(self):
+        print("Creating VPC and subnet")
+        response = self.ec2_client.create_vpc(
+            CidrBlock="10.0.0.0/16"
+        )
+        vpc_info = response.get("Vpc", {})
+        vpc_id = vpc_info.get("VpcId", {})
+        self.ec2_client.create_subnet(
+            CidrBlock="10.0.2.0/24",
+            VpcId=vpc_id
+        )
+        print("Created VPC and subnet")
+
+    def get_subnet_id(self):
+        print("Getting subnet ID")
+        response = self.ec2_client.describe_subnets()
+        subnets = response.get("Subnets", [{}])
+        subnet_id = subnets[0].get("SubnetId", "")
+        print(f"Got subnet ID {subnet_id}")
+        return subnet_id
+
+    def create_compute_environment(self, security_group_id, batch_service_role_arn, subnet_id):
+        print(f"Creating compute environment for security group {security_group_id}")
+        BATCH_COMPUTE_RESOURCES["securityGroupIds"].append(security_group_id)
+        BATCH_COMPUTE_RESOURCES["subnets"].append(subnet_id)
+        self.batch_client.create_compute_environment(
+            computeEnvironmentName=BATCH_COMPUTE_ENVIRONMENT_NAME,
+            type=BATCH_COMPUTE_ENVIRONMENT_TYPE,
+            computeResources=BATCH_COMPUTE_RESOURCES,
+            serviceRole=batch_service_role_arn
+        )
+        print("Created security group")
+
+    def create_batch_job_queue(self, queue_name):
+        print(f"Creating job queue {queue_name}")
+        self.batch_client.create_job_queue(
+            jobQueueName=queue_name,
+            priority=1,
+            state="ENABLED",
+            computeEnvironmentOrder=BATCH_COMPUTE_ENVIRONMENT_ORDER
+        )
+        print("Created job queue")
+
+    def create_s3_bucket(self, bucket_name):
+        print(f"Creating bucket {bucket_name}")
+        self.s3_client.create_bucket(
+            Bucket=bucket_name
+        )
+        print("Created bucket")
+
+    def setup_ses(self, email_address):
+        print(f"Verifying email address {email_address}")
+        self.ses_client.verify_email_identity(
+            EmailAddress=email_address
+        )
+        print("Verified email address")
+
+
+def wait_for_localstack():
+    healthcheck_url = "".join([LOCALSTACK_URL, "/health"])
+    counter = 0
+    while True and counter < 15:
```

Comment:
same as `while counter < 15`

Reply:
whoops, forgot to remove that during cleanup. good catch.